app/views/public_profile/show.html.erb

```diff
@@ -7,71 +7,202 @@
     background-repeat: no-repeat;
     background-size: cover;
   }
+  .user_field {
+    padding-top: 25px;
+    line-height: 10pt;
+  }
+  .user_field > h1 {
+    color: white;
+    font-size: 20pt;
+    font-weight: bold;
+    text-shadow: 0 0 0.35714em rgba(0, 0, 0, 0.5);
+  }
+  .user_field > h2 {
+    color: white;
+    font-size: 13pt;
+  }
+  #resume h3 {
+    color: #8c1515;
+    font-size: 18px;
+  }
+  #resume hr {
+    width: 1.3em;
+    border-bottom: 0.15em solid #8c1515;
+    margin: 0.5em 0;
+  }
+
 </style>
 
 <div class='col-sm-12'>
   <div class='card'>
     <div class="card-header">
-      <a href="https://placeholder.com"><img class="img-responsive" src="http://via.placeholder.com/150x150"></a>
-      <div class="card-header-actions">
-        <%= link_to(profile_vcard_path(format: 'vcf'), class: 'btn btn-sm btn-light') do %>
-          <%= fa_icon('id-card', text: 'vCard') %>
-        <% end %>
-        <%= link_to(root_path, class: 'btn btn-sm btn-light') do %>
-          <%= fa_icon('id-card', text: 'ORCID') %>
-        <% end %>
+      <div class="row">
+        <div class="col-lg-2 col-md-3 col-sm-4 col-xs-12" style="text-align: right;">
+          <a href="https://placeholder.com"><img class="img-fluid" src="https://loremflickr.com/150/150"></a>
+        </div>
+        <div class="col-lg-8 col-md-7 col-sm-8 col-xs-12">
+          <div class="user_field">
+            <h1>
+              <%= @identity.name %>
+            </h1>
+            <h2><%= @user.title %></h2>
+            <p class="social">
+              <a class="btn btn-brand btn-sm btn-twitter" target="_blank" href="https://www.twitter.com/<%= @user.twitter %>" type="button" style="margin-bottom: 4px">
+                <i class="fa fa-twitter"></i>
+              </a>
+              <a class="btn btn-brand btn-sm btn-linkedin" target="_blank" href="https://www.linkedin.com/<%= @user.linkedin %>" type="button" style="margin-bottom: 4px">
+                <i class="fa fa-linkedin"></i>
+              </a>
+              <a class="btn btn-brand btn-sm btn-html5" target="_blank" href="http://<%= @user.profile_preferences['website'] %>" rel="nofollow" type="button" style="margin-bottom: 4px">
+                <i class="fa fa-globe"></i>
+              </a>
+              <a class="btn btn-brand btn-sm btn-twitter" target="_blank" href="https://www.skype.com/<%= @user.skype %>" type="button" style="margin-bottom: 4px">
+                <i class="fa fa-skype"></i>
+              </a>
+            </p>
+          </div>
+        </div>
+        <div class="col-md-2 col-sm-12 col-xs-12">
+          <div class="card-header-actions">
+            <%= link_to(profile_vcard_path(format: 'vcf'), class: 'btn btn-sm btn-light') do %>
```

Comment:
vcard ile orcid'in yerlerini değiştirelim, öncelikli olan(vcard) sağda daha olsun